src/website/src/pages/ec/components/list/docs/usage.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  Lists group together related pieces of information so they are clearly
+  associated with each other and easy to read. Lists are good from a structural
+  point of view as they help create a well-structured and more accessible
+  content. Semantically correct lists help visitors read your web site.
+</Paragraph>
+
+## Anatomy
+
+Lists or "typography lists" are optimized for reading comprehension.
+
+A list consists of a single continuous column of subdivisions called rows that contain items.
+
+Content of each item may be plain text, or other elements (links, paragraphs, ...)
+
+A list should have a contextual coherence and allow the user to understand the message instantly.
```

Comment:
```suggestion
```

Reply:
Why would we remove this? this is helpful content